lib/plugin.js

```diff
@@ -6,7 +6,7 @@ const attachmentIDRegex = /\/\w+\(.*ID=([0-9a-fA-F-]{36})/;
 
 cds.on("loaded", function unfoldModel(csn) {
   if (!("Attachments" in csn.definitions)) return;
-  const csnCopy = structuredClone(csn)
+  const csnCopy = structuredClone(csn);
```

Comment:
Can you remove all the formatting changes in this file, as these lines are out of scope for this specific task.